configure.ac

```diff
@@ -916,6 +916,20 @@ cat > conftest.c <<EOF
         hppa-linux-gnu
 # elif defined(__ia64__)
         ia64-linux-gnu
+# elif defined(__loongarch__)
+#  if defined(__loongarch64)
```

Comment:
We recently discovered this symbol is in fact deprecated. Please use the more expressive `__loongarch_lp64` instead.